### PR TITLE
Add the desktop version of my portfolio

### PR DESCRIPTION
**Pull request by Diana-Duya**
- clone the repo to my local computer.
- use media queries to design the desktop version from mobile version
- add images
- modify html and css

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -48,112 +54,156 @@ <h2>I’am a Software Developer</h2>
...
-        </p>
-        <ul>
-            <li class="html2">html</li>
-            <li class="bootstrap2">bootstrap</li>
-            <li class="Ruby2">ruby</li>
-        </ul>        
+    <div class="works">
+        <div class="recent">
+            <div class="title">
+                <h2 class="first">My Recent Works</h2>
```
Comment (ichala) [inline index.html]: - [ ] kindly increase the size of this `h2` to match figma design 

**_your design :_**
image


**_figma design :_**
image

File: index.html [hunk 2]
```diff
@@ -48,112 +54,156 @@ <h2>I’am a Software Developer</h2>
...
+                <div class="languages">
+                    <ul>
+                        <li class="css">css<a href="#"></a></li>
+                        <li class="html">html<a href="#"></a></li>
+                        <li class="bootstrap">bootstrap<a href="#"></a></li>
+                        <li class="ruby">ruby<a href="#"></a></li>
+                        </ul>
+                </div>
+                <div><button  type="button" class="first-btn">See Project</button></div>
+              </div>
```
Comment (ichala) [inline index.html:86]: - [ ] The description of each of these projects should be aligned to the bottom of the image. kindly fix that

Figma design          |  your design
:-------------------------:|:-------------------------:
image  |  image

File: index.html [hunk 3]
```diff
@@ -48,112 +54,156 @@ <h2>I’am a Software Developer</h2>
...
+                    </div>
+                    <div class="style-grid">
+                        <div class="profession profession2">
+                            <h4 class="pro">Profesional Art Printing Data</h4>
+                            <p class="paragraph2">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+                            </p>
+                            <ul>
+                                <li class="html2">html</li>
+                                <li class="bootstrap2">bootstrap</li>
+                                <li class="Ruby2">ruby</li>
```
Comment (ichala) [inline index.html]: - [ ] Please note that the `See Project` show up only if the user hovers on the card, so basically all the cards should be like the default style below and only when the user hover the dark background get removed and the button appears

Default style            |  On hover
:-------------------------:|:-------------------------:
image |  image

File: index.html [hunk 4]
```diff
@@ -48,112 +54,156 @@ <h2>I’am a Software Developer</h2>
...
+            <h3 class="me">About me</h3>
+                <p class="info">Hello I’m a software developer! I can help you build a<br> product , feature or website Look through some of my work<br> and experience! If you like what you see and have a project<br> you need coded, don’t hestiate to contact me.
+                </p>
+            <div class="resume">
+                <button class="last-btn" type="button">Get My Resume</button>
+            </div>
+        </div>
+        <div class="line-cont">
+            <hr class="second-line">
+        </div>
```
Comment (ichala) [inline index.html]: - [ ] kindly increase the width of the `About me` div to match the figma design

image

File: index.html [hunk 5]
```diff
@@ -197,38 +245,41 @@ <h3 class="text1">Language</h3>
...
+            <hr class="hor">
+            <div class="email">
+                <input type="email" name="email" id="email" required placeholder="Email address">
+            </div>
+            <hr class="hor">
+            <div class="textarea">
+                <textarea name="message" id="message" cols="40" rows="6" required maxlength="500">Enter text here</textarea>
+            </div>
+            <div class="button-cont">
+                <button type="submit"  class="buttonl">Get in touch</button>
```
Comment (ichala) [inline index.html:263]: - [ ] kindly add the hover , focus, and click styling to all of your buttons following the Figma design 

image

File: index.html [hunk 6]
```diff
@@ -48,112 +54,156 @@ <h2>I’am a Software Developer</h2>
...
+                            <h4 class="pro">Profesional Art Printing Data</h4>
+                            <p class="paragraph2">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+                            </p>
+                            <ul>
+                                <li class="html2">html</li>
+                                <li class="bootstrap2">bootstrap</li>
+                                <li class="Ruby2">ruby</li>
+                            </ul>        
+                        </div>
+                        <div class="action-btn"><button class="second-btn" type="button">See Project</button></div>
```
Comment (ichala) [inline index.html:170]: - [ ] [OPTIONAL] i would suggest that you add a `cursor: pointer;` style to all of your button to change the cursor type when users hover on the buttons, that way your buttons will be more elegant and user friendly

Before            |  After
:-------------------------:|:-------------------------:
G1  |  G2

**Review by ichala — CHANGES_REQUESTED** [review]
Hi @Diana-Duya  , 

Good job so far! 👏👏👏 
There are some issues that you still need to work on to go to the next project but you're almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- No linters errors✅ 
- Good Design ✅ 

 ## Required Changes ♻️

- [ ] _**check the comments below**_


## Optional suggestions

- [ ] _**check the comments below**_

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Diana-Duya** [comment]
Hello @ichala 

- I made all the changes on the pages requested
- I also implemented the hover and focus  effect on the buttons

**Review by CollinsTatang — APPROVED** [review]
# Approved 👏  
### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**